modules/aggregation-layers/src/common/aggregator/gpu-aggregator/webgl-bin-sorter.ts

```diff
@@ -105,17 +105,7 @@ export class WebGLBinSorter {
       model.setVertexCount(props.vertexCount);
     }
     if (props.moduleSettings) {
-      const {viewport, devicePixelRatio, modelMatrix, coordinateSystem, coordinateOrigin} =
-        props.moduleSettings;
-      model.shaderInputs.setProps({
-        project: {
-          viewport,
-          devicePixelRatio,
-          modelMatrix,
-          coordinateSystem,
-          coordinateOrigin
-        }
-      });
+      model.shaderInputs.setProps(props.moduleSettings);
```

Comment:
- Would be nice to follow up with a renaming of this property (we can offer backwards compatiblity if desired)? Maybe - `moduleProps` or `shaderModuleProps`?